src/commands/init.js

```diff
@@ -0,0 +1,23 @@
+const { flags } = require('@oclif/command');
+const AbstractAuthenticatedCommand = require('../abstract-authenticated-command');
+const logger = require('../services/logger');
+
+class InitCommand extends AbstractAuthenticatedCommand {
+  async runIfAuthenticated() {
+    // TO BE REMOVED. JUST CHECKING IF THE FLAG PARSING IS WORKING HERE :)
+    const parsed = this.parse(InitCommand);
+    logger.info('working here 🤙 flag:', parsed.flags);
+  }
+}
+
+InitCommand.description = 'Set up your development environment in your current folder.';
+
+InitCommand.flags = {
```

Comment:
You are missing the `help` option and description when running `forest init --help` :)

Reply:
As discussed, to be consistent with other commands (and as it is not really useful as the help option is displayed only after having typed `--help`), let's not put it here. Do you agree?